tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/AspNetCore5IastTests.cs

```diff
@@ -742,7 +742,7 @@ public async Task TestDirectoryListingLeak(string featureEnvVar)
         newFixture.SetOutput(Output);
 
         var datetimeOffset = DateTimeOffset.UtcNow; // Catch vulnerability at the startup of the app
-        await TryStartApp(newFixture);
```

Comment:
Just wonder, should the _default_ be to set `new MockTracerAgent.AgentConfiguration { SpanMetaStructs = false }` - I feel like we're just waiting for some more flake somewhere else otherwise 😄

Reply:
Good question 💀, so far these two seem to be the tests where we `// Catch vulnerability at the startup of the app` hence why this flake is happening(we missed that test in the last PR), if we do see another flake I'll have to check if ASM wants to do a change and where. cc @e-n-0